build.gradle

```diff
@@ -106,6 +107,21 @@ task showDifferencesStrict(type: JavaExec) {
     classpath = sourceSets.test.runtimeClasspath
     args('test-data/oh.txt-result-strict', 'test-data/oh.txt-result-strict-temp', 'test-data/oh.txt', 'test-data/diff-strict.txt')
 }
+
+task individualTesting(type: JavaExec) {

```

Comment:
Both new tasks should be in the verification group or else they are rather hard to find in UIs that display tasks.